docs/api-reference.md

```diff
@@ -3,25 +3,623 @@ id: api-reference
 title: API Reference
 ---
 
-## Commands
+## Read Model Connector Interface
 
-### Command HTTP API
+The table below lists functions that a custom Read Model's connector should implement.
 
-A command can be sent using HTTP API.
+| Function Name             | Description                                                                                |
+| ------------------------- | ------------------------------------------------------------------------------------------ |
+| [connect](#connect)       | Initializes a connection to a storage.                                                     |
+| [disconnect](#disconnect) | Closes the storage connection.                                                             |
+| [drop](#drop)             | Removes the Read Model's data from storage.                                                |
+| [dispose](#dispose)       | Forcefully disposes all unmanaged resources that Read Models served by this connector use. |
```

Comment:
- 'Forcefully'? Why forcefully?
- "that Read Models served by this connector use" - This does not make any sense

Rewrite

Reply:
> * 'Forcefully'? Why forcefully?

Because it does not wait for any processes to finish. As in 'forcefully shutdown/reboot a computer'

Reply:
> * "that Read Models served by this connector use" - This does not make any sense

I simplified it as follows: "Forcefully disposes all unmanaged resources provided by this connector."

Reply:
'As in 'forcefully shutdown/reboot a computer' - You shut it down, or you do not.

Reply:
ok